keel-slack/src/main/kotlin/com/netflix/spinnaker/keel/slack/callbacks/ManualJudgmentCallbackHandler.kt

```diff
@@ -0,0 +1,132 @@
+package com.netflix.spinnaker.keel.slack.callbacks
+
+import com.netflix.spinnaker.keel.api.constraints.ConstraintStatus
+import com.netflix.spinnaker.keel.core.api.parseUID
+import com.netflix.spinnaker.keel.persistence.KeelRepository
+import com.netflix.spinnaker.keel.slack.SlackService
+import com.netflix.spinnaker.kork.exceptions.SystemException
+import com.slack.api.model.block.LayoutBlock
+import com.slack.api.model.kotlin_extension.block.withBlocks
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+import java.time.Clock
+import java.time.Instant
+
+/**
+ * This handler will handel slack callbacks coming from Manual Judgment notifications.
+ * First, it will update the constraint status based on the user response (either approve/reject)
+ * Second, it will construct an updated notification with the action preformed and the user how did it.
```

Comment:
```suggestion
 * This handler will handle slack callbacks coming from Manual Judgment notifications.
 * First, it will update the constraint status based on the user response (either approve/reject).
 * Second, it will construct an updated notification with the action preformed and the user who did it.
```